Makefile

```diff
@@ -121,8 +117,18 @@ gendoc:
 		xargs -I% md-to-godoc -input=%
 
 .PHONY: clean
-clean::
+clean:
```

Comment:
Have we thought about setting up convenient wrappers for makefile targets? We have an internal repo that teams use today and include in their Makefiles.